src/sentry/static/sentry/app/components/dropdownReact.jsx

```diff
@@ -133,13 +148,6 @@ class DropdownReact extends React.Component {
     }
   };
 
-  // Control whether we should hide dropdown menu when it is clicked
-  handleDropdownMenuClick = e => {
```

Comment:
why can this be deleted? slightly confused

Reply:
This code causes the dropdown to be immediately closed when any part of it is clicked (unless keepMenuOpen is `true`). It's non desirable behavior for nested dropdowns (e.g. https://github.com/getsentry/sentry/issues/6368).

Reply:
ah, thanks